src/main.js

```diff
@@ -43,7 +95,7 @@ function handleSelection({ id, value }) {
       break;
```

Comment:
Change this to 

async function handleSelection({ id, value }) {

Reply:
Okay so adding async allows us to use await?

Reply:
Yes, `async` allows you to use `await` within the function.
Any function declared with the `async` keyword in front of it will return a `Promise`.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function